topics/Routing_in_Ktor.md

```diff
@@ -122,6 +124,67 @@ To access a parameter value inside the route handler, use the `call.parameters`
 A path parameter with a tailcard (`{param...}`) matches all the rest of the URL path and puts multiple values for each path segment into parameters using `param` as a key. For example, `/user/{param...}` matches `/user/john/settings`.
 To access path segments' values inside the route handler, use `call.parameters.getAll("param")`. For the example above, the `getAll` function will return an array containing the _john_ and _settings_ values.
 
+### Regular expression {id="regular_expression"}
+
+Regular expressions can be used with all defining route handlers functions: `route`, `get`, `post`, and so on. 
+
+> To know more about regular expressions, see [Kotlin documentation](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/-regex/).
+
+Let's write a route matches any path that ends with `/hello`.
+
+```kotlin
+import io.ktor.server.routing.*
+import io.ktor.server.response.*
+
+routing {
+    get(Regex(".+/hello")) {
```

Comment:
It would be helpful to mention that a regex of a route is implicitly surrounded by `^` and `$`.

Reply:
It doesn't work for the route function. In next example `Regex(".+/hello")` wouldn't end with `$`:
```
route(Regex(".+/hello")) {
    get(Regex("/world") {
    ...
    }
}
```